server/services/interventionsService.ts

```diff
@@ -367,27 +354,27 @@ export default class InterventionsService {
     })) as PCCRegion[]
   }
 
-  async createDraftActionPlan(token: string, referralId: string): Promise<DraftActionPlan> {
+  async createDraftActionPlan(token: string, referralId: string): Promise<ActionPlan> {
     const restClient = this.createRestClient(token)
 
     try {
       return (await restClient.post({
         path: '/draft-action-plan',
         headers: { Accept: 'application/json' },
         data: { referralId },
-      })) as DraftActionPlan
+      })) as ActionPlan
     } catch (e) {
       throw this.createServiceError(e)
     }
   }
 
-  async getDraftActionPlan(token: string, actionPlanId: string): Promise<DraftActionPlan> {
+  async getActionPlanForReferral(token: string, referralId: string): Promise<ActionPlan> {
     const restClient = this.createRestClient(token)
 
     return (await restClient.get({
-      path: `/draft-action-plan/${actionPlanId}`,
+      path: `/action-plan?referralId=${referralId}`,
```

Comment:
Was this addition of `getActionPlanForReferral` here intentional? I don't see any added contract tests for it. Feels like a separate thing.

Reply:
this is likely a merge mistake i'll remove